### ISSUE TITLE
Automatically add "export PATH=$PATH:/path/to/komodo/bin" during the installation process to ~/.bashrc

### ISSUE THREAD

**Issue by Defman21** [Komodo/KomodoEdit#307]
This should be done automatically on Linux systems with bash/sh terminals.
The original question was [here](http://forum.komodoide.com/t/linux-how-do-i-open-komodo-edit-nightly-in-terminal/1601) but as usual I transformed it to a topic about a new feature :stuck_out_tongue_closed_eyes:

**Comment by Naatan** [comment]
I think the issue with doing this is not everyone would want you to be messing with their PATH. So it should probably be some type of prompt.

**Comment by Defman21** [comment]
Yep, that's a good idea. And the default answer should be "nope", yes? Like "Do you want to add the path to Komodo <IDE/Edit> executable to the PATH variable? [N/y]".
